exonum-java-binding/core/rust/build.rs

```diff
@@ -12,5 +13,8 @@ fn main() {
         println!("cargo:rustc-link-lib=dylib=c++");
     }
 
-    protobuf_generate("../src/main/proto", &["src/proto"], "protobuf_mod.rs");
+    ProtobufGenerator::with_mod_name("protobuf_mod.rs")
+        .with_input_dir("../src/main/proto")
+        .add_path("../src/main/proto")
```

Comment:
Do not even ask me about what is going on here and why we need two identical arguments for different methods.